src/bin/dolos/main.rs

```diff
@@ -186,7 +186,8 @@ impl Config {
     }
 }
 
-fn main() -> Result<()> {
+#[tokio::main]
+async fn main() -> Result<()> {
```

Comment:
opentelemetry requires that tokio is setup before it gets called, so I added it here

Reply:
I realize now that unfortunately this breaks `cargo run daemon` with the following error

```
thread 'main' panicked at /home/me/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.40.0/src/runtime/scheduler/multi_thread/mod.rs:86:9:
Cannot start a runtime from within a runtime. This happens because a function (like `block_on`) attempted to block the current thread while the thread is being used to drive asynchronous tasks.
```